src/app/work-item/work-item-quick-add/work-item-quick-add.component.scss

```diff
@@ -8,6 +8,15 @@
   border-top: 1px solid darken($color-pf-white, 20%);
   text-align: $textLeft;
   cursor: pointer;
+ 
+  .dropup {
+    .dropdown-menu {
+      i {
+        margin-top: 5px;
```

Comment:
if you add `width: 15px;` to this, it will align the text and provide equal spacing.
![screen shot 2017-04-19 at 3 54 30 pm](https://cloud.githubusercontent.com/assets/4032718/25199330/816039c8-2518-11e7-81e0-966d90b8ef5b.png)